requirements-dev.txt

```diff
@@ -1,8 +1,8 @@
 -r requirements.txt
+coverage==4.2
 flake8==2.5.0
 pytest==2.9.2
 mock==2.0.0
 pre-commit==0.7.6
-pytest-cov==2.2.0
```

Comment:
You weren't actually using pytest-cov (the pytest plugin for coverage).  coverage is the official project